<launch>
    <node pkg="sesame_navigation" type="detect_obstacles.py" name="obstacle_detector_uav_1" output="screen">
        <param name="body_frame" type="string" value="uav_1/base_link"/>
        <param name="local_frame" type="string" value="uav_1/odom"/>
        <param name="marker_size" type="double" value="0.1875"/>
        <param name="marker_id" type="int" value="0"/>
        <param name="publish_visualization" type="bool" value="true"/>
        <param name="viz_marker_size" type="double" value="0.25"/>
        <rosparam param="viz_marker_color">[1.0, 0.9, 0.0]</rosparam>

        <remap from="/mavros/local_position/pose" to="/uav_1/mavros/local_position/pose"/>
        <remap from="/camera/color/image_raw" to="/uav_1/camera/color/image_raw"/>
        <remap from="/camera/color/camera_info" to="/uav_1/camera/color/camera_info"/>
        <remap from="obstacles" to="/vision_stack/obstacles"/>
        <remap from="visualization_marker_array" to="/vision_stack/uav_1_obstacles_visualization"/>
    </node>
</launch>
